data/mappings/chibios-peripherals/stm32g474.hjson

```diff
@@ -0,0 +1,69 @@
+{
+  "split": {
+    "B9": {
+      "config.h": [
+        "#define SERIAL_USART_DRIVER SD3",
+        "#define SERIAL_USART_TX_PAL_MODE 7"
+      ],
+      "mcuconf.h": [
+        "#undef STM32_SERIAL_USE_USART3",
+        "#define STM32_SERIAL_USE_USART3 TRUE"
+      ],
+      "halconf.h": [
+        "#define HAL_USE_SERIAL TRUE"
+      ]
+    },
+    "B10/B11": {
+      "config.h": [
+        "#define SERIAL_USART_DRIVER SD3",
+        "#define SERIAL_USART_TX_PIN B10",
+        "#define SERIAL_USART_TX_PAL_MODE 7",
+        "#define SERIAL_USART_RX_PIN B11",
+        "#define SERIAL_USART_RX_PAL_MODE 7",
+        "#define SERIAL_USART_FULL_DUPLEX"
+      ],
+      "mcuconf.h": [
+        "#undef STM32_SERIAL_USE_USART3",
+        "#define STM32_SERIAL_USE_USART3 TRUE"
+      ],
+      "halconf.h": [
+        "#define HAL_USE_SERIAL TRUE"
+      ]
+    }
+  },
+  "backlight": {
+    "A7": {
+      "config.h": [
+        "#define BACKLIGHT_PWM_DRIVER PWMD17",
+        "#define BACKLIGHT_PWM_CHANNEL 1",
+        "#define BACKLIGHT_PAL_MODE 1"
+      ],
+      "mcuconf.h": [
+        "#undef STM32_PWM_USE_TIM17",
+        "#define STM32_PWM_USE_TIM17 TRUE"
+      ],
+      "halconf.h": [
+        "#define HAL_USE_PWM TRUE"
+      ]
+    }
+  },
+  "ws2812": {
+    "B2": {
+      "config.h": [
+        "#define WS2812_PWM_DRIVER PWMD20",
+        "#define WS2812_PWM_CHANNEL 1",
+        "#define WS2812_PWM_PAL_MODE 3",
+        "#define WS2812_PWM_DMA_STREAM STM32_DMA1_STREAM1",
+        "#define WS2812_PWM_DMA_CHANNEL 1",
+        "#define WS2812_PWM_DMAMUX_ID STM32_DMAMUX1_TIM20_UP"
```

Comment:
`WS2812_PWM_DMA_CHANNEL` should not be specified for this chip.

`WS2812_PWM_DMA_CHANNEL` refers to a form of request selection done by the DMA controller (some chips, like F4x1, have CHSEL bits in the DMA_SxCR register, some others, like F072, have a separate DMA_CSELR register; F103 and F303 don't have such selection and rely on special remapping registers in cases when the same DMA request can be handled by multiple streams, so `WS2812_PWM_DMA_CHANNEL` should not be defined for those chips too).  `WS2812_PWM_DMAMUX_ID` refers to the request selection done by the separate DMAMUX peripheral; if DMAMUX is present, the first form of request selection is not really needed, and probably is not implemented in any existing chips.